lib/trento/clusters/value_objects/hana_cluster_node.ex

```diff
@@ -17,6 +16,7 @@ defmodule Trento.Clusters.ValueObjects.HanaClusterNode do
   deftype do
     field :name, :string
     field :site, :string
+    # this attribute is deprecated, moved to the sites entry
```

Comment:
Like @rtorrero suggested previously, this attribute is not too concrete. Maybe just naming it would not harm. What do you think ?